dot/network/block_announce.go

```diff
@@ -103,7 +103,7 @@ func decodeBlockAnnounceMessage(r io.Reader) (NotificationsMessage, error) {
 // BlockAnnounceHandshake is exchanged by nodes that are beginning the BlockAnnounce protocol
 type BlockAnnounceHandshake struct {
 	Roles           byte
-	BestBlockNumber uint64
+	BestBlockNumber uint32
```

Comment:
Why did this change from uint64 to uint32?

Reply:
when I got the message from the kusama node it was 4 bytes for this field, not 8, will confirm with spec team